stories/documentation/forms/fieldset/fieldset-basic.stories.ts

```diff
@@ -0,0 +1,122 @@
+import { Meta, StoryFn } from '@storybook/angular';
+
+interface FieldsetBasicStory {
+	alignLeft: boolean;
+	mod: string;
+	helper?: string;
+	title: string;
+	content: string;
+}
+
+export default {
+	title: 'Documentation/Forms/Fieldset/Basic',
+	argTypes: {
+		alignLeft: {
+			description: 'Incompatible avec les mod toggle et switch.',
+			control: {
+				type: 'boolean',
+			},
+		},
+		mod: {
+			options: ['', 'mod-toggle', 'mod-switch'],
+			control: {
+				type: 'select',
+			},
+		},
+		helper: {
+			control: {
+				type: 'text',
+			},
+		},
+		title: {
+			control: {
+				type: 'text',
+			},
+		},
+		content: {
+			control: {
+				type: 'text',
+			},
+		},
+	},
+} as Meta;
+
+function getTemplate(args: FieldsetBasicStory): string {
+	const alignLeft = args.alignLeft ? 'mod-alignLeft' : '';
+	const mod = args.mod;
+	const helper = args.helper ? '<span class="fieldset-component-title-content-text-helper">{{ helper }}</span>' : '';
+	const title = args.title;
+	const content = args.content;
+
+	if (mod === 'mod-toggle')
+		return `
+		<div class="fieldset mod-toggle">
```

Comment:
Couldn't we get rid of this wrapper? 🤔

Reply:
I haven't found any other way to keep the code displayed clean. Or separate the stories. @Supamiu maybe?

Reply:
CSF3 should allow you to create a base story and generate a different one per mod, which would be displayed separately.
But this looks like it could really use a wrapper asap because if there's template changes based on mod, it means that the maintainability on it is quite bad.